db/migrate/20191107110337_add_accrediting_provider_to_courses.rb

```diff
@@ -0,0 +1,6 @@
+class AddAccreditingProviderToCourses < ActiveRecord::Migration[6.0]
+  def change
+    add_column :courses, :accrediting_provider_id, :integer
+    add_index :courses, %i[accrediting_provider_id code], unique: true
```

Comment:
Why do we need this index? Isn't `code` already unique?

Reply:
Yup that's right. I must of been on autopilot 👍